data/user_data_base.ps2

```diff
@@ -146,6 +147,16 @@ function New-ServerName {
     return $myName
 }
 
+function Check-AnyTagExists {
+    $tags = Get-EC2Tag -Filters @(@{Name="resource-id"; Values=$instanceId})
```

Comment:
group doesn't get created until this script runs.. so no. I was planning on using the aws:autoscaling:groupName, but I figured just wait until any tag is created.